apps/api/src/mf/chat/controllers/chat.controller.ts

```
@@ -0,0 +1,256 @@
+  }
+}
+
+// 使用withErrorHandler包装所有方法
+export const chatController = new ChatController()
+const methodNames = Object.getOwnPropertyNames(ChatController.prototype)
+  .filter(prop => prop !== 'constructor') as Array<keyof ChatController>
+
+methodNames.forEach(methodName => {
+  const originalMethod = (chatController[methodName] as Function).bind(chatController)
+    ; (chatController[methodName] as any) = withErrorHandler(originalMethod, methodName)
+})
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)